pkg/cmd/cli/restore/logs.go

```diff
@@ -35,15 +37,14 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "logs RESTORE",
 		Short: "Get restore logs",
+		Args:  cobra.ExactArgs(1),
 		Run: func(c *cobra.Command, args []string) {
-			if len(args) != 1 {
```

Comment:
@ncdc I've moved this logic to `Args:  cobra.ExactArgs(1)`, let me know if you still want this here.

Reply:
I didn't know that existed until I saw it in your #384. I'm fine with this, but I'd like to do it consistently for all of our commands that require a specific number of args.

Reply:
So, should I let this be, or change it for now to be consistent with the other commands?

Reply:
I'll file an issue

Reply:
#410